.evergreen/generated_configs/tasks.yml

```diff
@@ -1961,7 +1961,7 @@ tasks:
       - func: bootstrap-mongo-orchestration
       - func: run-mock-kms-servers
       - func: run-tests
-  - name: earthly-build-and-test
+  - name: earthly-build-check
```

Comment:
Confirming, is the inclusion of this change intentional?

Reply:
Yes. I think the Evergreen config was not regenerated after the task was renamed in https://github.com/mongodb/mongo-c-driver/pull/1395/commits/6e6546b52140ba8e105872c93e948333a439b6f8.